README.md

```diff
@@ -834,13 +834,19 @@ On login, Dink can submit a character summary containing data that spans multipl
     "slayer": {
       "points": 2204,
       "streak": 1074
+    },
+    "pets": {
+      "11995": "Pet chaos elemental",
+      "13071": "Chompy chick"
```

Comment:
Is the key not an integer?

Reply:
JSON spec unfortunately forces them into strings